app/domain/authentication/validate_webservice_access.rb

```diff
@@ -60,12 +60,9 @@ def validate_user_is_defined
       end
 
       def validate_user_has_access
-        # Ensure user has access to the service
         has_access = user_role.allowed_to?(@privilege, webservice_resource)
         unless has_access
-          @logger.debug(Log::UserNotAuthorized
```

Comment:
was this removed b.c of repetition for DEBUG/ERROR logs? In other words, logged in debug and logged in error?

Reply:
yes